src/game/boe.fileio.cpp

```diff
@@ -59,6 +60,7 @@ void finish_load_party(){
 	bool town_restore = univ.party.town_num < 200;
 	bool in_scen = univ.party.scen_name.length() > 0;
 	
+	end_scenario = false;
```

Comment:
I elected to put this in `handle_victory()` instead, but can you think of any reason that would be insufficient?

Reply:
Yes, it is probably better.